Sources/Apollo/GraphQLResponse.swift

```diff
@@ -72,13 +75,16 @@ public final class GraphQLResponse<Data: GraphQLSelectionSet> {
       let data = try decode(selectionSet: Data.self,
                             from: dataEntry,
                             variables: variables)
+      let extensions = body["extensions"] as? JSONObject
```

Comment:
This should probably be outside the `if let` statement since theoretically someone could return stuff in `extensions` even if `data` is `nil`.

Reply:
Ah. Yeah, I was wondering if I was making the right choice there or not. I'll make an update ASAP. Thank you!